### ISSUE TITLE
[Feature request] Innerglow

### ISSUE THREAD

**Issue by dmun** [yshui/picom#824]
Hello, I want to propose a pretty simple effect that makes a window look way nicer in my opinion, it's basically an inner glow, but without the blurry part since that will make the window cluttered.

In the screenshot I created the effect in AwesomeWM using widgets, but I realized I was definitely not using AwesomeWM right and should probably be doing this in a compositor or something.

I would like to contribute to this project myself, but I have no idea where to start, so does anyone have some tips?
![image](https://user-images.githubusercontent.com/39461169/167309516-5fd1a14b-a03e-4de7-8a90-9b7ac4bc0bf6.png)

**Comment by LoganDark** [comment]
You're talking about the inner white border, right? I agree that this makes windows look like 90% better - kinda gives skeuomorphic vibes, and I still think Windows 7 / Aero Glass > all...

**Comment by dmun** [comment]
> You're talking about the inner white border, right? I agree that this makes windows look like 90% better - kinda gives skeuomorphic vibes, and I still think Windows 7 / Aero Glass > all...

Yeah exactly! It just makes a window pop out more and makes the difference between a window and wallpaper way more noticeable while being pretty as well.

**Comment by LoganDark** [comment]
![](https://user-images.githubusercontent.com/4723091/177038086-dd2beaf7-2c40-452b-a140-7cd7ac2e81f0.png)

macOS is the undisputed king of making stuff look good, and the "innerglow" is a huge part of that :p
